k8s/deployments.go

```diff
@@ -378,7 +378,11 @@ func newWelcomeContainer(app *models.App) (c *api.Container) {
 func newSlugRunnerContainer(app *models.App, slug string, storageType string) (c *api.Container) {
 	// creating runner container
 	c = newContainer(*app.Name, slugRunnerImage)
-	c.Args = []string{"start", "web"}
+	processType := processTypeWeb
+	if app.ProcessType != nil {
+		processType = *app.ProcessType
+	}
+	c.Args = []string{"start", processType}
```

Comment:
It is not, if we edit old apps and put the process type in the annotation this check is useless.
The question is: edit old apps or not? Not sure....I supposed we won't edit but editing is not a big deal either.